aspnetcore/fundamentals/logging/index/samples/3.x/MyMain/Program.cs

```diff
@@ -0,0 +1,197 @@
+#define EventLog // Default // Azure // Default // Scopes // FilterFunction // MinLevel // FilterCode // FilterFunction
```

Comment:
There's some weird indentation and newlines in this file, especially around `)`, that you might like to clean up a bit for display in the snippets.